README.md

```diff
@@ -1,3 +1,7 @@
+The README has yet to be written.
```

Comment:
We could leave this comment on the readme out I believe. It doesn't relate to setting up the website and I doubt we will forget to rewrite the README when the time comes.